packages/project-utils/bundling/function/buildFunction.js

```
@@ -54,4 +73,21 @@ module.exports = options => {
             resolve();
         });
     });
+
+    const handlerFile = fs.readFileSync(options.cwd + "/build/handler.js", {
+        encoding: "utf8",
+        flag: "r"
+    });
+    const includesGraphQl = handlerFile.includes("handler-graphql");
+
+    if (includesGraphQl) {
+        try {
+            await getLatestTelemetryCode;
+        } catch (err) {
+            console.log("Error loading telemtry code");
```

These should throw an error, and not do `console.log`.